global.json

```diff
@@ -4,7 +4,7 @@
   },
   "sdk": {
     "allowPrerelease": false,
-    "version": "8.0.300",
+    "version": "8.0.301",
```

Comment:
This only bumps the dotnet SDK version. Do we need to update the az types package version?

Reply:
The az types package version is defined here https://github.com/Azure/bicep/blob/6dd8bf75faa4620d29c51a604a19ff9dab421483/src/Bicep.Core/Bicep.Core.csproj#L44.

Reply:
Thanks for the catch